receiver/collectdreceiver/collectd.go

```diff
@@ -59,6 +59,13 @@ func (r *collectDRecord) protoTime() *timestamppb.Timestamp {
 	return timestamppb.New(ts)
 }
 
+func (r *collectDRecord) startTimestamp(mdType metricspb.MetricDescriptor_Type) *timestamppb.Timestamp {
```

Comment:
I think this function deserves a test of its own.

Reply:
@jpkrohling Thank you very much!! PTAL

I've added test cases for this method. The cases are quite trivial, however, *I think* we should have good coverage for the function as it is being exercised not only in the newly added cases, but also indirectly in `TestDecodeMetrics()`.